internal/otel/config/telemetry.go

```diff
@@ -15,7 +17,10 @@ const ( // supported trace propagators
 )
 
 // Telemetry returns our basic telemetry configuration.
-func Telemetry() telemetry.Config {
+func Telemetry(metricsPort int) telemetry.Config {
+	if metricsPort == 0 {
```

Comment:
Should we handle this when we init the providers? Since they pass in the metrics port as config?

Reply:
Yeah, let's try that